shimtest/mockstub.go

```diff
@@ -90,6 +90,11 @@ func (stub *MockStub) GetArgs() [][]byte {
 	return stub.args
 }
 
+// GetMSPID ...
+func (stub *MockStub) GetMSPID() (string, error) {
+	return "", errors.New("Not Implemented")
+}
+
```

Comment:
This is a sign that we did not make a compatible change. You've had to hit an implementation. Now think of all of the hundreds of others that may have implemented this contract...